Create ui-c8y-1020-24-0-add-support-for-module-less-plugins-6979.md (#2406)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1020-24-0-add-support-for-module-less-plugins-6979.md

---
date: ""

> title: add support for module less plugins (#6979)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Support module-less plugins
```

Reviewer: Plugins without models are now supported

Reviewer 2: ```suggestion
title: Plugins without modules are now supported
```

product_area: Application enablement & solutions
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-60605
version: 1020.24.0
---

> add support for module less plugins (#6979)

Author: The Claude AI suggested to use the following body:
```suggestion
To simplify plugin development and deployment, support for module-less plugins has been added. With this change, plugins no longer require a module to function, allowing them to be implemented and deployed more easily. This impacts plugin developers who can now create plugins without the overhead of a module, and users who can benefit from a wider variety of available plugins that are simpler to install and use.
```

> To simplify plugin development and deployment, support for module-less plugins has been added. With this change, plugins no longer require a module to function, allowing them to be implemented and deployed more easily. This impacts plugin developers who can now create plugins without the overhead of a module, and users who can benefit from a wider variety of available plugins that are simpler to install and use.

Reviewer 3: ```suggestion
To simplify plugin development and deployment, support for module-less plugins has been added. With this change, plugins no longer require a module to function, allowing to implement and deploy plugins more easily. This impacts plugin developers who can now create plugins without the overhead of a module, and users who can benefit from a wider variety of available plugins that are simpler to install and use.
```
